10up/classifai
# Add OpenAI Embeddings in the Classic Editor Support

### Description of the Change
This PR adds the ClassifAI metabox for the classic editor for OpenAI Embeddings support.

<img width="1695" alt="Screen Shot 2023-06-29 at 8 54 52 PM" src="https://github.com/10up/classifai/assets/3365507/bb50accb-8d2c-42b7-992d-4233b3f6c0e9">


Closes [#496](https://github.com/10up/classifai/issues/496)

### How to test the Change
- Set up an integration with OpenAI in Language Processing > OpenAI Embeddings
- Ensure the Classify content option is turned on and at least one Post type that has the block editor disabled.
- In whatever taxonomy you've selected, ensure you have one or more terms saved
- Create a new piece of content and publish that content (or match whatever post status option you selected)
- Ensure one or more terms gets auto-assigned


### Credits

Props @phpbits 


### Checklist:
